src/internal/dnd-controller/__mocks__/controller.ts

```diff
@@ -41,3 +41,5 @@ export const mockDraggable = {
 export function useDraggable() {
   return mockDraggable;
 }
+
+export const useDroppable = vi.fn() as any;
```

Comment:
Is this line needed in the latest revision?

Reply:
nope!